plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/my-subscriptions.tsx

```diff
@@ -97,7 +126,33 @@ export default function MySubscriptions(): JSX.Element {
 			label: __( 'Actions', 'woocommerce' ),
 		},
 	];
-	const subscriptionsAvailable: Array< Subscription > = [];
+	const subscriptionsAvailable: Array< Subscription > = subscriptions.filter(
+		( subscription: Subscription ) =>
+			! subscriptionsInstalled.includes( subscription )
+	);
+
+	const getStatus = ( subscription: Subscription ): string => {
+		// TODO add statuses for subscriptions
+		if ( subscription.product_key === '' ) {
+			return __( 'Not found', 'woocommerce' );
+		} else if ( subscription.expired ) {
+			return __( 'Expired', 'woocommerce' );
+		}
+		return __( 'Active', 'woocommerce' );
+	};
+
+	const getVersion = ( subscription: Subscription ): string => {
+		if ( subscription.local.version === subscription.version ) {
+			return subscription.local.version;
+		} else if ( subscription.local.version && subscription.version ) {
+			return subscription.local.version + ' > ' + subscription.version;
+		} else if ( subscription.version ) {
+			return subscription.version;
+		} else if ( subscription.local.version ) {
+			return subscription.local.version;
+		}
+		return '';
```

Comment:
Since we're returning right away, we don't need else. Although not sure the version without the 
`else` is any better :point_down: 

```suggestion
		if ( subscription.local.version === subscription.version ) {
			return subscription.local.version;
		} 
		
		if ( subscription.local.version && subscription.version ) {
			return subscription.local.version + ' > ' + subscription.version;
		}
		
		if ( subscription.version ) {
			return subscription.version;
		}
		
		  if ( subscription.local.version ) {
			return subscription.local.version;
		}
		
		return '';
```